deploy/helm/sumologic/values.yaml

```diff
@@ -4139,11 +4026,8 @@ metadata:
         pprof: {}
       exporters:
         sumologic/containers:
-          log_format: json
-          json_logs:
-            add_timestamp: true
-            timestamp_key: timestamp
-          endpoint: ${SUMO_ENDPOINT_DEFAULT_LOGS_SOURCE}
+          log_format: otlp
```

Comment:
`otlp` is default format for `logs`, `metrics`, `traces`

Reply:
I know, but I thought being explicit about this would make the PR easier to review.